app/views/hiring_staff/vacancies/vacancy_form_partials/_submit.html.haml

```diff
@@ -1,6 +1,14 @@
 - if @vacancy.published?
   = f.govuk_submit t('buttons.update_job')
 - else
-  = f.govuk_submit t('buttons.save_and_continue') do
-    %br
-    = link_to t('buttons.save_and_return'), jobs_with_type_school_path('draft'), class: 'govuk-link'    
+  - if f.class == GOVUKDesignSystemFormBuilder::FormBuilder
+    %div.continue.no-margin-bottom
```

Comment:
No need for `%div`

Reply:
I need to apply styling to the button, but the form builder abstracts away from it such that you can't apply classes or ids directly to the button.